common/gnome-shell/3.30/sass/_common.scss

```diff
@@ -899,7 +899,9 @@ StScrollBar {
   background-gradient-direction: none;
   background-color: transparent;
   border-bottom-width: 0;
-  border-image: url('common-assets/panel/panel.svg') 1 1 1 1;
+  border-image: none;
+
+	&.solid { border-image: url('common-assets/panel/panel.svg') 1 1 1 1; }
```

Comment:
The indention is wrong here, right? Saw this in the file above as well.

Reply:
Is it possible that you are missing the `#panel.solid {` part?

Reply:
> Is it possible that you are missing the `#panel.solid {` part?

The "&" selector should be the correct selector for the scss syntax, and will be replaced by "#panel" in the css stylesheet